Scripts/extras/pull_and_build_from_git.py

```diff
@@ -619,8 +621,14 @@ def build_and_install_cmake(self):
                                       '-S', str(cmakelists_folder.absolute()),
                                       '-B', str(self.build_folder.name)]
 
-                if self.custom_toolchain_file:
-                    cmake_generator_args.append( f'-DCMAKE_TOOLCHAIN_FILE="{self.custom_toolchain_file}"')
+                if self.package_info.custom_toolchain_file:
+                    custom_toolchain_file = self.package_info.custom_toolchain_file
+                    custom_toolchain_file_path = pathlib.Path(custom_toolchain_file).absolute().resolve()
```

Comment:
The [resolve()](https://docs.python.org/3/library/pathlib.html?highlight=pathlib#pathlib.Path.resolve) method makes the path absolute. 
Therefore the `absolute()` call is redundant

Reply:
I'll remove it